docs/strategy-advanced.md

```diff
@@ -69,6 +69,20 @@ class AwesomeStrategy(IStrategy):
 
 See `custom_stoploss` examples below on how to access the saved dataframe columns
 
+## Custom sell signal
+
+It is possible to define custom sell signals. This is very useful when we need to customize sell conditions for each individual trade.
+
+An example of how we can sell trades that were open longer than 1 day:
+
+``` python
+class AwesomeStrategy(IStrategy):
+    def custom_sell(self, pair: str, trade: Trade, current_time: datetime, current_rate: float,
+                    current_profit: float, **kwargs) -> bool:
+        time_delta = datetime.datetime.utcnow() - trade.open_date_utc
```

Comment:
you should definitely not use "datetime.datetime.now()" here - but current_time.

Otherwise you'll break backtesting ... 

I'd however make this example ... a bit better (this example can be done by setting ROI of -1 too).
Maybe we can check for trades > 1 day that are in the red currently (freeing up a negative trade slot) - while staying in trades that are positive (riding the profits ...) ?